lib/tsconfig/4.8.x/ng15/README.md

```diff
@@ -0,0 +1,20 @@
+<h1 align="center">
+   code style - TypeScript 4.8.x + Angular Angular 15
+</h1>
+
+## About
+
+This sub-package hosts the [TypeScript](https://www.typescriptlang.org/) configuration for TypeScript 4.8.x. and Angular 15
```

Comment:
Only the version of typescript >= 4.8.2 are compatible with Angular 15. 
Could you adapt to `for TypeScript >= 4.8.2. and Angular 15` ?